kvmagent/kvmagent/plugins/host_plugin.py

```
@@ -1090,12 +1088,12 @@ def start_write_to_server(self):
         pkt_counter = 0
         while True:
             try:
-                self.host_socket.send(str(pkt_counter))
+                self.host_socket.send(str(pkt_counter).encode())
             except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)